src/style.css

```diff
@@ -1,3 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  color: #000;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-width:1024px;
+  width: 100%;
+  background: #fff;
+  flex-shrink: 0;
+  padding: 0 50px;
+}
+
+.link {
+  margin-right: 20px;
+  height: 60px;
+  line-height: 60px;
+  font-weight: 500;
+}
+
+.link:hover {
+  color: #00ACDC;
+}
+
+.link:last-child{
+margin-right: 0;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```

Comment:
it's redundant

Reply:
I had been doing this with one of your teachers , it passed tests . After 'fixing' with your suggestions the **** sends me errors again ... i would rather leave it unsolved , then spend another week doing  nothing .